cal.py

```
@@ -0,0 +1,10 @@
+
+from constants import *
+
+def check(num):
+    if num == A_constant:
```

Ruff (0.8.2)

`A_constant` may be undefined, or defined from star imports

(F405)